packages/core/src/components/BaseInput/BaseInput.tsx

```diff
@@ -2,11 +2,14 @@ import { useContext } from "react";
 
 import {
   InputBaseComponentProps as MuiInputBaseComponentProps,
-  InputProps as MuiInputProps,
-  Input as MuiInput,
```

Comment:
Can't we get rid of the other two? It looks like we wouldn't be losing anything
`InputBaseComponentProps` is just a `HTMLAttributes<HTMLInputElement | HTMLTextAreaElement>`
and `InputBaseProps` is just used for the ref